static/scripts/rewards/render-transaction/render-transaction.ts

```diff
@@ -8,14 +8,19 @@ import { insertErc20PermitTableData, insertErc721PermitTableData } from "./inser
 import { renderEnsName } from "./render-ens-name";
 import { renderNftSymbol, renderTokenSymbol } from "./render-token-symbol";
 import { ERC20Permit, Permit, TokenType } from "@ubiquibot/permit-generation/types";
+import { displayCommitHash } from "../init";
 
 const carousel = document.getElementById("carousel") as Element;
+const footer = document.querySelector(".footer") as Element;
 const table = document.querySelector(`table`) as HTMLTableElement;
 type Success = boolean;
 
 export async function renderTransaction(): Promise<Success> {
+  displayCommitHash(); // @DEV: display commit hash in footer
+
   if (app.claims && app.claims.length > 1) {
     carousel.className = "flex";
+    footer.classList.add("animate");
```

Comment:
This should be in the main body of the function so that it happens regardless of the amount of permits to claim